src/midi/midi_coreaudio.h

```diff
@@ -76,9 +76,10 @@ class MidiHandler_coreaudio final : public MidiHandler {
 	          soundfont(nullptr)
 	{}
 
-	std::string_view GetName() const override
+	const std::string& GetName() const override
 	{
-		return "coreaudio";
+		static const std::string name("coreaudio");
+		return name;
 	}
```

Comment:
I don't like this pattern.  Most of the time I would avoid return values being references.  It's kind of annoying because this is where `std::string_view` should be useful except we interface with C APIs in the usage code.

I would just return a `std::string` (doesn't need to be const) and be done with it though.  It's not performance sensitive code plus return value optimization and small string optimization should both apply here anyway (automatically done by a compiler in a release build).

Reply:
True and true. I'll fix it.

Reply:
100% agree @weirddan455.